### ISSUE TITLE
Does not persist across devices

### ISSUE THREAD

**Issue by jenswachtel** [orbitdb/orbit-db#228]
I was experimenting with this cool db. 

It seems that I am connected
e.g. console: "Swarm listening on /libp2p-webrtc-star/dns4/star-signal.cloud.ipfs.team/wss/ipfs/QmTcaqq1eP44LwTqc6eVfXDP9iy5aVzaSsdVpyeUxdaBEU"

I can run db.put commands, it's definitely persisted. If I restart my application and do a db.load waiting for event 'ready', I can then read those previously stored documents.

I was curious and also set up a "synced" event, which never gets called. Can this event take more than 5mins to be called? (I am talking about 3 rows with 2 fields as a test)

Because if I go to a different device and start my application, do a db.load waiting for event 'ready' and then query, I don't receive any results. 

Not sure If I missed anything? My understanding was that this will "eventually" get synced/stored on IPFS and thus if I open e.g. orbitdb.docstore('orbit.jens.dummyServices') I shall receive it again from anywhere. (I do read content on IPFS is stored with no guarantee unless I pin it) But I didn't read anything about pinning this orbitdb?

I would be happy to get help on this question. Maybe I missed a crucial part that I couldn't find in the github documentation

Thanks,
Jens

**Comment by niksmac** [comment]
I remember facing the same issue which I managed to get through by 

```
db.events.on('ready', () => {
  console.log(db.get('{YOURQUERY}'))
})
db.events.on('synced', () => {
  console.log(db.get('{YOURQUERY}'))
})
db.load()
```

Looks like `db.load()` is missing from the docs. 

Event `synced` will get called on the other machine than the one you are making the `write`. Create another instance of the same db on a different machine to test this.

**Comment by haadcode** [comment]
Thanks for reporting the issue @jenswachtel and for helping out @niksmac!

We just released a new version [v0.18.0](https://www.npmjs.com/package/orbit-db) and it contains many fixes and improvements. Specifically for [replication](https://github.com/orbitdb/orbit-db/blob/master/CHANGELOG.md#replication) and [persistency](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md#persistency). Check out also the new [getting started guide](https://github.com/orbitdb/orbit-db/blob/master/GUIDE.md) which contains more information on usage and how OrbitDB works, as well as plenty of examples.

Persistency and replication should be fixed in the new release, so closing this. However, if you still have problems, please let us know and re-open the issue!
